src/staging/bot.py

```diff
@@ -1515,9 +1515,7 @@ async def configure_devel_bci_package(self, package_name: str) -> None:
                 + (", ".join(b.package_name for b in self._bcis))
             )
 
-        assert (
-            len(bci) == 1
-        ), f"Got {len(bci)} packages with the name {package_name}: {bci}"
+        assert len(bci) >= 1, f"Got no packages with the name {package_name}: {bci}"
```

Comment:
This will now be an empty list:
```suggestion
        assert len(bci) >= 1, f"Got no packages with the name {package_name}"
```